Code/Tools/FBuild/FBuildCore/Graph/NodeGraph.cpp

```diff
@@ -523,6 +523,13 @@ Node * NodeGraph::GetNodeByIndex( uint32_t index ) const
 	return n;
 }
 
```

Comment:
If you move the DisplayTargetList function into NodeGraph, you won't need to expose this.